nodejs/week1/meal-share/src/routes/meal.js

```diff
@@ -1,12 +1,12 @@
-const express = require('express');
-const app = express();
+const express = require("express");
 const fs = require("fs");
+const app = express();
+const mealData = fs.readFileSync(__dirname + "/../data/meals.json");
+const meals = JSON.parse(mealData);
+const reviewData = fs.readFileSync(__dirname + "/../data/reviews.json");
+const reviews = JSON.parse(reviewData);
 
-// Respond with the json for a random meal (including it's reviews)
-const meals = JSON.parse(fs.readFileSync(__dirname + "/data/meals.json"));
-const reviews = JSON.parse(fs.readFileSync(__dirname + "/data/reviews.json"));
-
-app.get ("/meal",function(req, res) {
+app.get ("/",(req, res) => {
     const random = Math.floor(Math.random()* meals.length);
     const ranmdoMeal = meals.filter(meal => meal.id ===random);
```

Comment:
`    const random = Math.floor(Math.random()* meals.length); `
this random can be a number from 0 to 3 in your case.

`    const randomMeal = meals.filter(meal => meal.id === random);`
and since you don't have any meal id=0, sometimes you will get empty array, and never a meal with id=3.

Either you need to add 1 at the end of the expression 
`    const random = Math.floor(Math.random()* meals.length) + 1; `

or there is shorter way to write these two lines    
`const randomMeal = meals[Math.floor(Math.random() * meals.length)];`